src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-topbar/entity-viewer-indicator/EntityViewerIndicator.tsx

```diff
@@ -16,22 +16,14 @@
 
 import React from 'react';
 
-import styles from './ScaleSwitcher.scss';
+import styles from './EntityViewerIndicator.scss';
 
-const ScaleSwitcher = () => {
+const EntityViewerIndicator = () => {
   return (
     <div className={styles.container}>
```

Comment:
Several things about this component:

1) Now that it's no longer a scale switcher, there is no point in having a `container` class. It should probably be just a single DOM element.
2) We may want to add a prop (let's say `{entity: string}`), which will be used to pass to the component the type of entity this is. So that we do not hard-code the word "Gene" inside the component, but make the component reusable (we already know it's going to be used in species selector)
3)  Class name `selectedNode` no longer makes sense. We should probably name the class of this single DOM element `entityIndicator`.
4) Could you please rename the component (and its scss counterpart) from EntityViewerIndicator to EntityIndicator? We already know this component is going to be used in species selector. We might even move this component to the `src/shared/components` folder.